2_process.R

```diff
@@ -16,6 +17,39 @@ p2_targets_list <- list(
                        sites = p1_drb_temp_sites_sf)
   ),
   
+  tar_target(p2_buffered_nhd_reaches,
+              st_buffer(p1_nhd_reaches, dist = 250)
```

Comment:
Since `p1_nhd_reaches` is in lat/lon coordinates, it seems safer to explicitly define the units of `dist`: 

```suggestion
              sf::st_buffer(p1_nhd_reaches, dist = units::set_units(250, m))
```

Reply:
I also noticed that the resulting buffers are pretty "_jaggedy_" (see below). I wonder if that's due to how buffers get calculated with lat/lon data and the s2 engine. We may need to transform the reaches to a projected crs before buffering (e.g. `sf::st_buffer(sf::st_transform(p1_nhd_reaches_along_NHM, 5070), dist = 250)`), or maybe turn off the s2 engine here as you did before. 

![weird_buffer](https://user-images.githubusercontent.com/8785034/184686800-5eb4a293-e074-444c-9507-0dfca1ec4930.png)